Makefile

```diff
@@ -0,0 +1,21 @@
+.PHONY: default release
+
+VERSION=$(shell git describe --tags)
+SHELL:=/bin/bash
+
+default:
+	@echo "See https://github.com/datawire/forge/blob/master/DEVELOPING.md"
+
+version:
+	@echo $(VERSION)
+
+## Setup dependencies ##
+
+virtualenv:
+	virtualenv --python=python2 virtualenv
+	virtualenv/bin/pip install -Ur requirements.txt
```

Comment:
I think this is redundant, the pip install -e . should install all the necessary dependencies automagically.